src/pages/PoolsPage/index.tsx

```diff
@@ -53,24 +52,22 @@ const PoolsPage = () => {
                 pool?.[0]?.[1]?.interior?.X2?.[1]?.GeneralIndex.replace(/[, ]/g, "")
               );
 
-              const assetTokenBalance = toUnit(
+              const assetTokenBalance = formatDecimalsFromToken(
                 poolReserve?.[1].replace(/[, ]/g, ""),
```

Comment:
This will crash if there is no key 1 in the array. U can not use replace on undefined. Add "?" after [1]